xml/soc-operations-gpu-passthrough.xml

```diff
@@ -0,0 +1,146 @@
+       already be enabled. To check if IOMMU is enabled:
+      </para>
+<screen>&prompt.root;virt-host-validate
+.....
+QEMU: Checking if IOMMU is enabled by kernel
+: WARN (IOMMU appears to be disabled in kernel. Add intel_iommu=on to kernel cmdline arguments)
+.....</screen>
+      <para>
+       To modify the kernel cmdline as suggested in the warning, edit the file
+       <filename>/etc/default/grub</filename> and append
+       <literal>intel_iommu=on</literal> to the
+       <literal>GRUB_CMDLINE_LINUX_DEFAULT</literal> variable. Then run
+       <literal>update-bootloader</literal>.
+      </para>
+      <para>
+       A reboot will be required for <literal>iommu_groups</literal> to be
+       enabled.
+      </para>
+     </step>
+     <step>
+      <para>
+       After the reboot, check that IOMMU is enabled:
+      </para>
+<screen>&prompt.root;virt-host-validate
+.....
+QEMU: Checking if IOMMU is enabled by kernel
+: PASS
+.....</screen>
+     </step>
+     <step>
+      <para>
+       Confirm IOMMU groups are available by finding the group associated with
+       your PCI device (for example Nvidia GPU):
+      </para>
+<screen>&prompt.ardana;lspci -nn | grep -i nvidia
+08:00.0 VGA compatible controller [0300]: NVIDIA Corporation GT218 [NVS 300] [10de:10d8] (rev a2)
+08:00.1 Audio device [0403]: NVIDIA Corporation High Definition Audio Controller [10de:0be3] (rev a1)</screen>
+      <para>
+       In this example, <literal>08:00.0</literal> and
+       <literal>08:00.1</literal> are addresses of the PCI device. The vendorID
+       is <literal>10de</literal>. The productIDs are <literal>10d8</literal>
+       and <literal>0be3</literal>.
+      </para>
+     </step>
+     <step>
+      <para>
+       Confirm that the devices are available for passthrough:
+      </para>
+<screen>&prompt.ardana;ls -ld /sys/kernel/iommu_groups/*/devices/*08:00.?/
+drwxr-xr-x 3 root root 0 Feb 14 13:05 /sys/kernel/iommu_groups/20/devices/0000:08:00.0/
+drwxr-xr-x 3 root root 0 Feb 19 16:09 /sys/kernel/iommu_groups/20/devices/0000:08:00.1/</screen>
+      <note>
+       <para>
+        With PCI passthrough, only an entire IOMMU group can be passed. Parts
+        of the group cannot be passed. In this example, the IOMMU group is
+        <literal>20</literal>.
+       </para>
+      </note>
+     </step>
+    </procedure>
+   </listitem>
+   <listitem>
+    <para>
+     <emphasis role="bold">Preparing &o_comp; and &o_img; for
+     passthrough</emphasis>
+    </para>
+    <para>
+      Information about configuring &o_comp; is available in the
+      documentation at
+      <link
+      xlink:href="https://docs.openstack.org/nova/rocky/admin/pci-passthrough.html"/>.
+      Services like <literal>nova-compute</literal>, <literal>nova-scheduler</literal>
+      and <literal>nova-api</literal> must be configured.
+     </para>
+     <note>
+       <para>
+         We suggest user to create new <filename>nova.conf</filename> file
```

Comment:
Add a bit more context here to help customer know the pros and cons of creating unversioned file.